README.md

```diff
@@ -1,12 +1,21 @@
-#This project was abandoned because [Moxie Marlinspike said he was not OK with LibreSignal using the Open Whisper Systems servers and the name "Signal"](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-217211165).
+#This GitHub repository has been abandoned.
```

Comment:
@xmikos So when would you say LibreSignal started (since it didn't start in February 2016)? Most people know LibreSignal as the websocket-based build of TextSecure/Signal. Would you agree if I wrote that LibreSignal started [in August 2015](https://twitter.com/xmikos/status/636494031464321024), you started distributing a build that was based on JavaJens's websocket repo?